internal/node/node.bzl

```diff
@@ -638,6 +641,7 @@ nodejs_binary_kwargs = {
     "implementation": _nodejs_binary_impl,
     "outputs": _NODEJS_EXECUTABLE_OUTPUTS,
     "toolchains": [
+        # what is the best way to resolve this
```

Comment:
is that a question for me?

Reply:
Sorry that comment slipped through. It's more relevant to a follow up pr